proto/v1beta1/swagger/project.swagger.json

```diff
@@ -21,7 +21,7 @@
         "operationId": "ListProjects",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```

Comment:
This file should be unchanged by the PR, right? If so, let's revert all the previous changes.